package.json

```diff
@@ -26,5 +25,8 @@
     "passport-local": "^1.0.0",
     "request": "^2.87.0",
     "underscore": "^1.9.1"
+  },
+  "devDependencies": {
+    "dotenv": "^6.2.0"
```

Comment:
Why did you move 'dotenv' as a devdependency?

Reply:
I felt since its only need in development environment, it should be devDependency. But I have moved it to dependencies regardless